pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
 requires = [
-  "setuptools>=19.6",
+  "setuptools>=59.0.0",
```

Comment:
I guess it's been a while since we updated this requirement, eh?

Reply:
Yep, 0.19 probably doesn't actually work anymore actually

Reply:
for posterity: setuptools 19.6 only supported Python <= 3.5, so it's clearly not viable anymore
https://pypi.org/project/setuptools/19.6/